install_files/ansible-base/roles/build-ossec-deb-pkg/tasks/main.yml

```diff
@@ -52,9 +52,10 @@
         SHA1: {{ ossec_download.stat.checksum }}
 
       Try rerunning the playbook to download the files again.
-  when: not (ossec_download.stat.exists and
-             ossec_download.stat.md5 == "{{ ossec_md5_checksum }}" and
-             ossec_download.stat.checksum == "{{ ossec_sha1_checksum }}")
+  when:
+    - not ossec_download.stat.exists
```

Comment:
This logic isn't the same. You need to add a `!=` for the other two to get the correct negations.